examples/multi-services/otel-collector-config.yaml

```diff
@@ -0,0 +1,44 @@
+# This is where we set up our OpenTelemetry collector
+
+receivers:
+  # setting up how we receive data from our services
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+      http:
+        endpoint: 0.0.0.0:4318
+
+processors:
+  # bundle up the data before sending it out
+  batch:
+
+exporters:
+  # send metrics to otlp
+  prometheus:
+    endpoint: "0.0.0.0:8889"
+    namespace: "otelchi"
+    const_labels:
+      label1: value1
```

Comment:
i was wondering what is this mean @fathiraz ?

Reply:
the const_labels in the `otel-collector-config.yaml` is a configuration option for the prometheus exporter that allows to add constant labels to all metrics exported to prometheus, this means that every metric exported to prometheus will have an additional label label1 with the value value1. this is useful when you want to:
- add environment information (e.g., environment: production)
- add cluster information (e.g., cluster: us-west)

i've changed it to make it more useful :
```yaml
const_labels:
      environment: development
      cluster: local
```